lib/bolt/transport/winrm.rb

```diff
@@ -84,7 +84,7 @@ def run_task(target, task, arguments, _options = {})
 
           if ENVIRONMENT_METHODS.include?(input_method)
             arguments.each do |(arg, val)|
-              cmd = "[Environment]::SetEnvironmentVariable('PT_#{arg}', '#{val}')"
+              cmd = "[Environment]::SetEnvironmentVariable('PT_#{arg}', @'\n#{val}\n'@)"
```

Comment:
```
PS /Users/Iristyle/source/puppetlabs-dsc/pkg> [Environment]::SetEnvironmentVariable('PT_foo', @'                   >> some val                                                                                                        >> asd;f                                                                                                           >> '@)
```

Yields


```
PS /Users/Iristyle/source/puppetlabs-dsc/pkg> $env:PT_foo
some val
asd;f
```

👍

Reply:
Where do we want to document the limitations of this?